create_gz_vendor_pkg/create_vendor_package.py

```diff
@@ -364,7 +362,7 @@ def main(argv=sys.argv[1:]):
         "--overwrite_cmake_configs",
```

Comment:
we may want to rename this argument since it doesn't write cmake configs anymore, though it does control writing of dsv files

Reply:
Technically those are cmake config files (files ending `.in` which are supposed to be consumed by cmake's `configure`)